### PR TITLE
Issue100 Types

### PR DESCRIPTION
**Pull request by Jonas-K-S**
Kayla please check the models for types and naming.

### CODE REVIEW

File: src/models/facility.model.ts [hunk 1]
```diff
@@ -29,10 +29,10 @@ const FacilitySchema: Schema = new Schema({
   Operations: { type: String, required: true },
   Reviews: [{ type: mongoose.Schema.Types.ObjectId, ref: "Review" }],
   Date: { type: Date, required: true },
-  PictureURL: { type: String, required: false },
+  PictureURL: { type: [String], required: false },
   RatingAVG: { type: Number, required: true },
   Favorites: { type: Number, required: true },
```
Comment (kaylantran) [inline src/models/facility.model.ts]: Shouldn't Favorites be false as well especially as when a facility is first created it wouldn't be favorited by any?

File: src/models/review.model.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ interface IReview extends Document {
   Rating: number;
   Likes: number;
   Dislikes: number;
-  PictureURL: string;
+  PictureURL: string[];
   FacilityID: mongoose.Schema.Types.ObjectId;
```
Comment (kaylantran) [inline src/models/review.model.ts:9]: I think either FacilityID or UserId needs should change so they match -- i.e. UserID or FacilityId?

**Review by kaylantran — CHANGES_REQUESTED** [review]
It is good overall; however, now that these changes are made -- you need to mirror these changes in the controller functions as they expect the previous types

File: src/models/review.model.ts (hunk 2 above)
Comment (Jonas-K-S) [reply src/models/review.model.ts:9]: what do you mean

**Review by kaylantran — APPROVED** [review]
Good!

**Pull request merged**